solutions.js

```diff
@@ -1,5 +1,65 @@
+//HW INSTRUCTS
+
+/*
+Tonight's training involves taking over imgur.com with javascript!
+You will need to write a function for each of the following requests below.
+Once you have successfully tested the function in the browser, copy and save your code
+in solutions.js with an appropriate functionName.
+*/
+
 console.log('Defining functions!');
 
+//1) Create a function that changes the navbar to a different color.
 function changeNavColor() {
   // change the navbar to a different color
+  var navbar = document.querySelector('div#topbar');
+  navbar.style['background-color'] = 'teal'; // ex 2
+  console.log(navbar);
 };
+
+changeNavColor();
+
+
+//2) Add a new <h1> to the top of the page that says.
+function addNewH1(){
+   var myText = "<h1>This site is now under Jesse Lewis's control!</h1>";
+   var Element = document.querySelector('.sentence-sorting');
```

Comment:
Whenever you are creating a variable you'll want to keep it lowercase. Uppercase letters works but it signifies a "class" in JS - which is something we haven't talked about yet.